attributes/default.rb

```diff
@@ -9,11 +9,11 @@
 when 'ubuntu'
   default['ceph']['init_style'] = 'upstart'
 when 'rhel', 'centos'
-  if node['platform_version'] >= 7
+  if node['platform_version'].to_f >= 7
```

Comment:
Previously I was comparing to version 7.2 so this is a leftover. Shouldn't cause any problems.